target/elc.c

```diff
@@ -120,11 +129,20 @@ int main(int argc, char* argv[]) {
   Module* module = load_eir(stdin);
 #else
   target_func_t target_func = NULL;
+  const char* ext = NULL;
   const char* filename = NULL;
   for (int i = 1; i < argc; i++) {
     const char* arg = argv[i];
     if (arg[0] == '-') {
-      target_func = get_target_func(arg + 1);
+      if (target_func) {
+        handle_args_func_t handle_args = get_handle_args_func(ext);
+        if (!handle_args || !handle_args(arg, argv[++i])) {
```

Comment:
Since `arg` is always going to start with a dash, wouldn't it be easier to use `arg + 1` here so that the handle args function doesn't have to think about the dash?

Reply:
Thanks for the suggestion! Done.